### ISSUE TITLE
Persiapan Training Model

### ISSUE THREAD

**Issue by nurul274** [fadilahnurkh09/intprog2-project#19]
Menyiapkan konfigurasi dan data yang diperlukan sebelum proses training model MobileNetV2 untuk klasifikasi bentuk wajah.

- [ ] Memastikan dataset training siap digunakan
- [ ]  Memastikan dataset validation siap digunakan
- [ ]  Menentukan ukuran input 224 × 224 × 3
- [ ]  Menentukan batch size
- [ ]  Menentukan jumlah epoch
- [ ]  Menentukan optimizer
- [ ]  Menentukan learning rate
- [ ]  Menentukan loss function
- [ ]  Menentukan metrik accuracy
